src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2674,7 +2674,7 @@ const openDetectedLinkNoKb = nls.localize('openDetectedLinkNoKb', 'The Open Dete
 const newWithProfile = nls.localize('newWithProfile', 'The Create New Terminal (With Profile) ({0}) command allows for easy terminal creation using a specific profile.');
 const newWithProfileNoKb = nls.localize('newWithProfileNoKb', 'The Create New Terminal (With Profile) command allows for easy terminal creation using a specific profile and is currently not triggerable by a keybinding.');
 const accessibilitySettings = nls.localize('accessibilitySettings', 'Access accessibility settings such as `terminal.integrated.tabFocusMode` via the Preferences: Open Accessibility Settings command.');
-
+const commandPrompt = nls.localize('commandPromptMigration', "Consider using powershell instead of command prompt for an improved experience");
```

Comment:
While this is true, considering that this was proposed because all content was not in the accessible buffer which it now will be, is this too much noise for users who want to use command prompt?

Reply:
IMO, users will rarely access terminal help as it is. Shell integration will still be a much better experience because we will have the symbol provider. Based on what JooYoung said, screen reader users primarily use command prompt. It's worth mentioning in this menu as their experience could be improved - in particular since we'll now provide all content so pwsh won't mess with things

Reply:
My bad, I thought this was in the aria label 🤦